vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -146,6 +221,24 @@ void ScrollViewShadowNode::updateProperties(winrt::Microsoft::ReactNative::JSVal
         m_isScrollingEnabled = scrollEnabled;
         SetScrollMode(scrollViewer);
       }
+    } else if (propertyName == "isFlatList") {
+      const auto [valid, isFlatList] = getPropertyAndValidity(propertyValue, true);
+      if (valid && isFlatList) {
+        // We use the XAMl ScrollViewer for both the RN ScrollView and FlatList - need arrow navigation of FlatList
+        // entries.
+        m_PreviewKeyDownRevoker = scrollViewer.PreviewKeyDown(
+            winrt::auto_revoke, [=](const winrt::IInspectable &sender, const xaml::Input::KeyRoutedEventArgs &e) {
+              OnPreviewKeyDown(sender, e);
+            });
+        m_GettingFocusRevoker = scrollViewer.GettingFocus(
+            winrt::auto_revoke, [=](const winrt::IInspectable &sender, const xaml::Input::GettingFocusEventArgs &e) {
+              OnGettingFocus(sender, e);
+            });
+        m_LosingFocusRevoker = scrollViewer.LosingFocus(
+            winrt::auto_revoke, [=](const winrt::IInspectable &sender, const xaml::Input::LosingFocusEventArgs &e) {
+              OnLosingFocus(sender, e);
```

Comment:
you can omit the types of the lambdas and just do `const auto&` since it's not important you're just forwarding